### ISSUE TITLE
TypeError: listEventsCalendar.map is not a function

### ISSUE THREAD

**Issue by sachowdhary786** [jrsousadev/fullcalendar-scheduler#5]
When attempting to run the project, I come across an issue where it states that listEventsCalendar.map is not a function in the EventCalendar.ts file

**Comment by krdq-source** [comment]
Obviously, the problem is that you haven't deployed the backend. Go to` src/pages/index.tsx` file and in line `36 `replace 
 `const listAllEventsCalendar = mapArrayEventCalendar(eventsCalendar);` to `const listAllEventsCalendar = (typeof eventsCalendar === 'function') ? mapArrayEventCalendar(eventsCalendar) : null;`
